src/wallet/wallet.cpp

```diff
@@ -511,12 +511,29 @@ void CWallet::SetBestChain(const CBlockLocator& loc)
 std::set<std::pair<libzcash::PaymentAddress, uint256>> CWallet::GetNullifiersForAddresses(const std::set<libzcash::PaymentAddress> & addresses)
 {
     std::set<std::pair<libzcash::PaymentAddress, uint256>> nullifierSet;
+    // Sapling ivk -> addr map
+    std::map<libzcash::SaplingIncomingViewingKey, libzcash::SaplingPaymentAddress> ivkMap;
+    for (const auto & addr : addresses) {
+      if (boost::get<libzcash::SaplingPaymentAddress>(&addr) != nullptr) {
```

Comment:
In many other places as we have been adding sapling support we have been removing calls to `boost::get<..>(...)` and replacing it with a `boost::static_visitor` of some sort. If the `get` is necessary here, we may want to consider adding a visitor to handle getting the nullifiers.